wiremock/stubs/pdus.json

```diff
@@ -150,7 +150,7 @@
     "serviceScope": "temporary-accommodation"
   },
   {
-    "id": "f6fd8e4d-50a4-438d-83b8-335c538b8b66",
+    "id": "f6fd8e4d-50a4-438d-83b8-335c538b8b67",
```

Comment:
This should solve this absolutely random test failure 😄  https://github.com/ministryofjustice/hmpps-temporary-accommodation-ui/actions/runs/10669075094/job/29570513513#step:7:553

Reply:
is there anyway we can make this human friendly reference instead of a "magic number"? It will make it a lot easier to spot issues like incorrect region/pdu.

Reply:
These are stubs so replicate database data; we could theoretically use more human-friendly ids but then we're deviating from what the DB returns, which probably isn't desirable either? After this change, there are no longer any PDU with a duplicated id in this stub:

```js
const pdus = require('./wiremock/stubs/pdus.json')
pdus.map(x => x.id).length === [...new Set(pdus.map(x => x.id))].length // returns true
```